playwright/tests/meeting/agenda.spec.js

```diff
@@ -275,7 +275,7 @@ test.describe('past - desktop', () => {
             const eventButtons = row.locator('.agenda-table-cell-links > .agenda-table-cell-links-buttons')
             if (event.flags.agenda) {
               // Show meeting materials button
-              await expect(eventButtons.locator('i.bi.bi-collection')).toBeVisible()
+              await expect(eventButtons.locator(`#btn-btn-${event.id}-meeting-materials`)).toBeVisible()
```

Comment:
btn-btn-?

Reply:
Yep, just following the convention as link ids are `btn-lnk-${item.id}-tar` so I made button ids `btn-btn-${event.id}-meeting-materials`